src/main/java/org/aksw/simba/lemming/metrics/single/nodetriangles/ayz/ListingAyzMetric.java

```diff
@@ -34,32 +35,38 @@ public ListingAyzMetric() {
 
     @Override
     public double apply(ColouredGraph graph) {
-        double delta = 3.0;
-        double threshold = Math.pow(graph.getGraph().getNumberOfEdges(), (delta - 1) / (delta + 1));
-        IntSet highDegreeVertices = new DefaultIntSet(Constants.DEFAULT_SIZE); 
-        		//IntSets.from();
+
+        double r = 2.376;
+        double threshold = Math.pow(graph.getGraph().getNumberOfEdges(), (r - 1) / (r + 1));
+        IntSet highDegreeVertices = new DefaultIntSet(Constants.DEFAULT_SIZE);
+        IntSet lowDegreeVertices = new DefaultIntSet(Constants.DEFAULT_SIZE);
```

Comment:
It might be better to initialize both sets with the maximum size they may have (i.e., the number of vertices of the graph).

Reply:
Done.